.github/ISSUE_TEMPLATE/4-release.yml

```
@@ -0,0 +1,125 @@
+name: 'WooCommerce: release checklist'
+description: Create a new issue to track the next WooCommerce core release progress.
+title: "WooCommerce RELEASE_VERSION: RC, final and dot-releases progress tracking"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        This issue provides visibility on the progress of the release process of WooCommerce core RELEASE_VERSION and 
+        centralizes any conversations about it. The ultimate goal of this issue is to keep a reference of the steps, 
+        resources, work, and conversations about this release so it can be helpful for the next contributors releasing 
+        a new WooCommerce core version.
+
+        -   WooCommerce core version to release: RELEASE_VERSION ([milestone](https://github.com/woocommerce/woocommerce/milestone/MILESTONE_ID))
+        -   Release lead: RELEASE_LEAD
+        -   RC release date: RELEASE_DATE
+        -   Final release date: RELEASE_DATE
+        -   WooCommerce core change log: [here](https://github.com/woocommerce/woocommerce/blob/trunk/changelog.txt)
+
+  - type: markdown
+    attributes:
+      value: |
+        ## Resources
+
+        -   [Release calendar](https://developer.woocommerce.com/release-calendar/)
+        -   [Previous releases](https://developer.woocommerce.com/releases/)
+        -   Latest (version 9.9+) [release process changes](https://developer.woocommerce.com/2025/03/05/upcoming-changes-to-the-woocommerce-release-process/)
+        -   Internal detailed reference to the release process: Pf59Ax-7F-p2
+
+  - type: markdown
+    attributes:
+      value: |
+        ## Checklist
+
+  - type: markdown
+    attributes:
+      value: |
+        ### RC release
+        
+        #### Get release artifact ready
+
+        -    [] Ensure minimally required WordPress and PHP versions are up-to-date
+        -    [] Ensure all [CFEs](https://github.com/woocommerce/woocommerce/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22code%20freeze%20exception%22) for the target release version are addressed and merged
+        -    [] Validate stable tag in `readme.txt` for the release branch and update if necessary
```

Can we be more explicit on what "validation" means and what we're updating to?

```suggestion
        -    [] Validate that the stable tag in `readme.txt` in the release branch matches RELEASE_VERSION or update if necessary
```

---

Updated in f0100792c3917d6d53b68f76b41f52c15a5e6d57

---

Hold on. The stable tag is getting updated later in the process, so we can not use RELEASE_VERSION in the checklist.

We can drop manual validation steps in artifacts build section once https://github.com/woocommerce/woocommerce/pull/58241 is merged.

---

As long as it not merged, WDYT about this change: 8066c29fd6f93eae6e310f56b01daba8f6cc1086 ?

---

That looks great for now.  Just to double check: these validation items will be removed anyway once https://github.com/woocommerce/woocommerce/pull/58241 is merged?

---

Yep!

---

Dropping the entry for manually checking stable tags before building zips as #58241 is merged.